catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -117,18 +118,37 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
 
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
```

Comment:
besides loadTable, should we process the other DDL?  createTable, dropTable, AlterTable, etc

Reply:
I have checked it, others did not use `DatabaseMetaData`, so it's ok, you can check it.